spring/boot2-actuator-autoconfigure/src/test/resources/application-basePathSamePortTest.yml

```diff
@@ -0,0 +1,21 @@
+# Use-legacy-processing to use this application-testbed.yml file instead of application.yml
+spring:
+  config:
+    use-legacy-processing: true
+
+armeria:
+  ports:
+    - port: 0
+  internal-services:
+    include: metrics, health, actuator
+    port: 10000
```

Comment:
wanna test internal-server.port == management.server.port. But this may make test flaky

Reply:
Haven't tried but can we set the port programmatically like this?
https://stackoverflow.com/questions/29072628/how-can-i-override-spring-boot-application-properties-programmatically

Reply:
Or we can just pick a random number that you like 😄  and use `@FlakyTest`.